coalib/misc/CheckVersion.py

```diff
@@ -0,0 +1,16 @@
+from pkg_resources import get_distribution, DistributionNotFound
+
+
+def check_version(package_name, version):
+    """
+    Check if a particular package version is present or not.
+
+    :param package_name: Package name.
+    :param version:      Version number to match.
+    :return:             ``True`` if package is with given ``version``,
```

Comment:
is **installed** with ;)

Reply:
ahh, really sorry for annoying you with my lack of alertness :-1:   Pushed the changes